packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -30,6 +32,8 @@ export enum RelaySubmitType {
 }
 
 export class Relayer {
+  private readonly l2BlockFinder: BlockFinder<BlockTransactionBase>;
```

Comment:
why does this have to be defined with this `BlockTransactionBase`? @mrice32 's PR had the same type declaration; why is this required?